src/main.rs

```diff
@@ -44,12 +57,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
                 get_vade(&sub_m)?.did_resolve(&did).await?
             }
             ("update", Some(sub_m)) => {
-                let did = get_argument_value(&sub_m, "did", None);
-                let options = get_argument_value(&sub_m, "options", None);
-                let payload = get_argument_value(&sub_m, "payload", None);
-                get_vade(&sub_m)?
-                    .did_update(&did, &options, &payload)
-                    .await?
+                wrap_vade3!(did_update, sub_m)
```

Comment:
Is this still working?

left one has arguments `did`, `options` and `payload`
right macro passes arguments `method`, `options` and `payload` to vade